src/Table/Table.js

```diff
@@ -167,15 +184,18 @@ export default class Table extends React.Component {
 
   getRowCells(columns, sortBy, buildRowOptions, idAttribute, row) {
     let id = row[idAttribute];
-    // Skip build row if we have it memoized
-    if (this.cachedRows[id] == null) {
-      let rowCells = columns.map(function (column, index) {
-        // For each column in the data, output a cell in each row with the value
-        // specified by the data prop.
-        let cellClassName = getClassName(column, sortBy, row, columns);
 
-        let cellValue = row[column.prop];
+    let rowCells = columns.map((column, index) => {
+      let cellClassName = getClassName(column, sortBy, row, columns.length);
+      let cellValue = row[column.prop];
+      let cellID;
 
+      if (!column.dontCache) {
+        cellID = this.buildUniqueID(cellClassName, cellValue, column);
```

Comment:
Maybe we could just pass down `column.prop`, and since we are using `column.prop` multiple places let's create a variable for it.